### PR TITLE
[#5964] hardwallet screens flow 2

### LINKED ISSUE #5964: Build screens for HW Flow 2
### Description

Most of the screens for hardwallet flow 2 (`Exporting the client account from a logged-in state to the hardwallet`) are the same as flow 1 screens, but there are some new that needs to be built:

* Enter pair code https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows?node-id=591%3A998
* Screens from step 3 which are the same, but different text and button ("initiated but not paired", "no slots available", "card is already linked to account") https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows?node-id=587%3A44

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows">Figma</a></strong></div><div>Created with Figma</div></blockquote>
<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### PR DESCRIPTION
**Pull request by dmitryn**
fixes #5964

### Summary:

Screens for hardwallet flow 2 https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows?node-id=0%3A1

### Review notes (optional):
Screens are hidden behind flag. 
HW Flows are work-in-progress, probably would change in the future.

status: ready

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/UfQjpWl1hmRchHIyY3Wvu2nW/Hardwallet-light-flows">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### CODE REVIEW

File: src/status_im/ui/screens/hardwallet/setup/styles.cljs [hunk 1]
```diff
@@ -106,11 +148,13 @@
...
   {:font-size   15
+   :width       "90%"
+   :text-align  :center
    :padding-top 5
    :color       colors/gray})
 
-(def puk-code-numbers-container
-  {:width           369
+(defn puk-code-numbers-container [width]
+  {:width           (* 0.9 width)
```
Comment (rasom) [inline src/status_im/ui/screens/hardwallet/setup/styles.cljs]: Is that parent's `width`? Can't it be just `"90%"` then?

File: src/status_im/ui/screens/hardwallet/setup/views.cljs [hunk 2]
```diff
@@ -83,38 +70,130 @@
...
+     (i18n/label :t/estimated-time
+                 {:time estimated-time-text})]]
+   [react/view styles/waiting-indicator-container
+    [react/activity-indicator {:animating true
+                               :size      :large}]]])
+
+(defn preparing []
+  [loading-view {:title-label         :t/preparing-card
+                 :text-label          :t/generating-codes-for-pairing
+                 :estimated-time-text "~20 seconds"}])
```
Comment (goranjovic) [inline src/status_im/ui/screens/hardwallet/setup/views.cljs]: externalize i18n strings

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/hardwallet/setup/styles.cljs (hunk 1 above)
Comment (dmitryn) [reply src/status_im/ui/screens/hardwallet/setup/styles.cljs]: @rasom `width` is a window width here. For some reason "90%" gives more width than `(* 0.9 width)` (check "puk code" on 2nd screenshot, it's getting too close to the edge. 1st screenshot shows `(* 0.9 width)` solution )

screenshot 2018-10-02 13 27 11
screenshot 2018-10-02 13 28 02

Comment (dmitryn) [reply src/status_im/ui/screens/hardwallet/setup/styles.cljs]: I was trying "90%" value at first, but it didn't work so i switched to dynamic calculation to not spend too much time on researching why it doesn't work.

File: src/status_im/ui/screens/hardwallet/setup/views.cljs (hunk 2 above)
Comment (dmitryn) [reply src/status_im/ui/screens/hardwallet/setup/views.cljs]: good point, done.

File: src/status_im/ui/screens/hardwallet/setup/styles.cljs (hunk 1 above)
Comment (dmitryn) [reply src/status_im/ui/screens/hardwallet/setup/styles.cljs]: @rasom do you see any issues with calculating width based on window width?

Comment (dmitryn) [reply src/status_im/ui/screens/hardwallet/setup/styles.cljs]: Removed dynamic width calculation based on window width.

Comment (rasom) [reply src/status_im/ui/screens/hardwallet/setup/styles.cljs]: I don't see any issue, just suggesting

**Review by rasom — APPROVED** [review]

**Pull request merged**